tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/validation/PersonValidator.java

```diff
@@ -66,15 +71,35 @@ public PersonValidator(PersonRepository personRepository, ReferenceService refer
    * Custom validation on the personDto DTO, this is meant to supplement the annotation based
    * validation already in place. It checks that the public health number unique check if it is
    * entered.
+   * Validation for update doesn't validate the values which are not changed.
    *
-   * @param personDto the person to check
+   * @param personDto      the person to check
+   * @param originalDto    the original person to update if validation type is Update,
+   *                       can be set to null if validation type is Create
+   * @param validationType the validation type
    * @throws MethodArgumentNotValidException if there are validation errors
    */
-  public void validate(PersonDTO personDto) throws MethodArgumentNotValidException {
-    List<FieldError> fieldErrors = new ArrayList<>();
-    fieldErrors.addAll(checkMandatoryFields(personDto));
-    fieldErrors.addAll(checkRole(personDto));
-    fieldErrors.addAll(checkPublicHealthNumber(personDto));
+  public void validate(PersonDTO personDto, PersonDTO originalDto, Class<?> validationType)
+      throws MethodArgumentNotValidException {
+    List<FieldError> fieldErrors = new ArrayList<>(checkMandatoryFields(personDto));
```

Comment:
I think this is fine but want to check we've agreed/let the Data "PO" know we are still check the new `personDto` has all fields rather than the merged object having all the mandatory fields.